src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -338,6 +337,13 @@ else if (args.indexOf('\"') != -1)
         return newL;
     }
 
+    public static byte [] arrayFromArrayWithLength(byte[] array, int length) {
```

Comment:
This method `arrayFromArrayWithLength()` has a truly **awful** name -- can we name it something more descriptive like `padArrayToLength`? Also, public method needs javadoc.

Actually, is it even necessary to repeat elements from the original array in wraparound fashion using modulus? Can you just delete this method and replace it with calls to `Arrays.copyOf(byte[] original, int newLength)`, which pads with 0's?